tests/e2e/testcases/blockchain/configure/suite.go

```diff
@@ -0,0 +1,470 @@
+				"node-config": "invalidPath",
+			},
+		)
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+		gomega.Expect(output).Should(gomega.ContainSubstring("open invalidPath: no such file or directory"))
+	})
+
+	ginkgo.It("check default configs", func() {
+		output, err := utils.TestCommand(
+			utils.BlockchainCmd,
+			"deploy",
+			[]string{utils.BlockchainName},
+			utils.GlobalFlags{},
+			utils.TestFlags{
+				"local":             true,
+				"skip-icm-deploy":   true,
+				"skip-update-check": true,
+			},
+		)
+		gomega.Expect(output).Should(gomega.ContainSubstring("L1 is successfully deployed on Local Network"))
+		gomega.Expect(err).Should(gomega.BeNil())
+		blockchainID, err := utils.ParseBlockchainIDFromOutput(output)
+		gomega.Expect(err).Should(gomega.BeNil())
+		nodesInfo, err := utils.GetLocalClusterNodesInfo()
+		gomega.Expect(err).Should(gomega.BeNil())
+		checkBlockchainConfig(nodesInfo, blockchainID, defaultRPCTxFeeCap, nil)
+		checkSubnetConfig(nodesInfo, "")
+		checkNodeConfig(nodesInfo, -1)
+	})
+
+	ginkgo.It("set blockchain config", func() {
+		// set blockchain config before deploy
+		chainConfig := getBlockchainConfig(newRPCTxFeeCap1)
+		chainConfigPath, err := utils.CreateTmpFile(constants.ChainConfigFileName, []byte(chainConfig))
+		gomega.Expect(err).Should(gomega.BeNil())
+		_, err = commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"chain-config": chainConfigPath,
+			},
+		)
+		gomega.Expect(err).Should(gomega.BeNil())
+		// deploy l1
+		output, err := utils.TestCommand(
+			utils.BlockchainCmd,
+			"deploy",
+			[]string{utils.BlockchainName},
+			utils.GlobalFlags{},
+			utils.TestFlags{
+				"local":             true,
+				"skip-icm-deploy":   true,
+				"skip-update-check": true,
+			},
+		)
+		gomega.Expect(output).Should(gomega.ContainSubstring("L1 is successfully deployed on Local Network"))
+		gomega.Expect(err).Should(gomega.BeNil())
+		blockchainID, err := utils.ParseBlockchainIDFromOutput(output)
+		gomega.Expect(err).Should(gomega.BeNil())
+		nodesInfo, err := utils.GetLocalClusterNodesInfo()
+		gomega.Expect(err).Should(gomega.BeNil())
+		// check
+		checkBlockchainConfig(nodesInfo, blockchainID, newRPCTxFeeCap1, nil)
+		// stop
+		err = commands.StopNetwork()
+		gomega.Expect(err).Should(gomega.BeNil())
+		// cleanup logs
+		cleanupLogs(nodesInfo, blockchainID)
+		// change blockchain config
+		chainConfig = getBlockchainConfig(newRPCTxFeeCap2)
+		chainConfigPath, err = utils.CreateTmpFile(constants.ChainConfigFileName, []byte(chainConfig))
+		gomega.Expect(err).Should(gomega.BeNil())
+		_, err = commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"chain-config": chainConfigPath,
+			},
+		)
+		gomega.Expect(err).Should(gomega.BeNil())
+		// start
+		out := commands.StartNetwork()
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network ready to use"))
+		// check
+		checkBlockchainConfig(nodesInfo, blockchainID, newRPCTxFeeCap2, nil)
```

Comment:
instead of checking through local files whether this value has been changed, can we check it through an api call to local network instead?

Reply:
prefer to keep checks based on avalanchego logs, as no API call was found to do checks for each config, after some investigation. unless you provide a config field and API call to use

Reply:
@artemanosov indeed each test check that config settings are applied during both deploy and restart. 
as they use the same blockchain, I preferred to use the same test case. do you suggest to use different 
tests?